ntbs-service/wwwroot/css/testResults.scss

```diff
@@ -0,0 +1,21 @@
+@import "./breakpoints";
+@import "./colors.scss";
+
+
+.notification-lab-results-summary {
+    padding: 10px 5px 10px 5px;
+    border: 1px;

```

Comment:
Stylistic choice here but might be nice to be explicit with border colour etc. and use the shorthand `border: 1px solid <colour>`